### ISSUE TITLE
iFrame embedding breaks when disable 3rd party cookies is enabled

### ISSUE THREAD

**Issue by nkconnor** [metabase/metabase#10970]
Embedded dashboard tries to access localStorage; which when in an iFrame (3rd party), and user has 3rd party cookies disabled, throws 

```
Uncaught DOMException: Failed to read the 'localStorage' property from 'Window': Access is denied for this document.
```

At the least it would be helpful to display an error message indicating why the embed fails. But with 3rd party cookie blocking becoming popular, I think it makes more sense to allow embedding without writing to local storage.

**To Reproduce**
Publish dashboard, embed in an iFrame, disable 3rd party cookies in Chrome, and view.

Tested on multiple versions of Chrome and OS, including Version 76.0.3809.132 (Official Build) (64-bit), MacOSX & Windows.

**Comment by flamber** [comment]
Hi @nkconnor 
Sounds like a duplicate of #8873, which was fixed by #10533, which was included since 0.33.2

**Comment by nkconnor** [comment]
Yes I think you are right. Will test and close afterward.

**Comment by salsakran** [comment]
Hello, @nkconnor not sure if you've had a chance to test this.

Since you and flamber both think this was fixed by #10533, I'm going to close this. If it wasn't please chime in, and we'll reopen this and more importantly get to the bottom of this.
